tests/test_inference.py

```diff
@@ -5296,6 +5305,36 @@ def test_duplicated_keyword_arguments(self) -> None:
         site = self._call_site_from_call(ast_node)
         self.assertIn("f", site.duplicated_keywords)
 
+    def test_call_site_uninferable(self) -> None:
+        code = """
+            def get_nums():
+                nums = ()
+                if x == '1':
+                    nums = (1, 2)
+                return nums
+
+            def add(x, y):
+                return x + y
+
+            nums = get_nums()
+
+            kwargs = {foo: bar, 1: baz}
```

Comment:
I think you need to introduce some ambiguity by assigning `kwargs = {}` before reassigning it to have content, like you did with `nums`.